core/src/block.rs

```diff
@@ -711,23 +711,23 @@ impl From<SignedBlock> for VersionedCandidateBlock {
 declare_versioned_with_scale!(VersionedCommittedBlock 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema, Serialize);
 
 impl VersionedCommittedBlock {
-    /// Convert from `&VersionedCommittedBlock` to V1 reference
+    /// Convert to V1 reference
     #[inline]
     pub const fn as_v1(&self) -> &CommittedBlock {
         match self {
             Self::V1(v1) => v1,
         }
     }
 
-    /// Convert from `&mut VersionedCommittedBlock` to V1 mutable reference
+    /// Convert to V1 mutable reference
```

Comment:
These are self-explanatory and frankly not very well-documented. I'd be fine with `#[allow(missing_docs)]`

Reply:
Should I remove all edited comments of this kind and replace them with `#[allow(missing_docs)]`?

Reply:
Yep

Reply:
Done.